gcc/jit/libgccjit.h

```diff
@@ -1999,6 +1999,56 @@ gcc_jit_vector_type_get_element_type (gcc_jit_vector_type *vector_type);
 extern gcc_jit_type *
 gcc_jit_type_unqualified (gcc_jit_type *type);
 
+#define LIBGCCJIT_HAVE_ATTRIBUTES
+
+/* Function attributes.  */
+enum gcc_jit_fn_attribute
+{
+  GCC_JIT_FN_ATTRIBUTE_ALIAS,
+  GCC_JIT_FN_ATTRIBUTE_ALWAYS_INLINE,
+  GCC_JIT_FN_ATTRIBUTE_INLINE,
```

Comment:
What did we decide to do for the `inline` keyword?

Reply:
We're didn't decide anything as far as I can remember.

Reply:
Ok, so let's ask David if he has an idea when sending the patch.